.github/workflows/ci.yml

```diff
@@ -18,8 +18,12 @@ jobs:
     - uses: actions/checkout@v4
     - name: Update system
       run: sudo apt update -y
-    - name: Install dependencies
-      run: sudo apt install -y git gcc cmake make postgresql postgresql-server-dev-all
+    - name: Install postgresql
+      run: |
+        sudo apt install -y postgresql-14
```

Comment:
No 13 ?

Reply:
If we use version 13, we have to install it from the source code as it cannot be installed through the repository.